directord/interface.py

```diff
@@ -92,3 +92,4 @@ def __init__(self, args):
                 connection_string=self.connection_string,
                 interface=self,
             )
+            self.thread(target=self.driver.run, daemon=False).start()
```

Comment:
this isn't going to work. the interface class isn't initialized when driver.run starts responding to messages.